### PR TITLE
Final (probably) update

### PR DESCRIPTION
**Pull request by ivxandx**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,9 +7,1023 @@
...
+            <span class="name">FISHIES</span><span style="color: white"> (7.28%)</span></p>
+          </div>
+         <div class="nominee">
+          <img src="Media/Emote/Normgasm.webp" loading="lazy">
+          <p>
+            <span class="number">#2: </span>
+            <span class="name">NormGasm</span><span style="color: white"> (21.46%)</span></p>
+         </div>
+         <div class="nominee">
+          <img src="Media/Emote/CINEMA.webp" loading="lazy">
```
Comment (vyneer) [inline index.html:483]: a few emotes don't seem to load for me, seems like CINEMA, for example, is a jpg in the files, but here u reference a webp

**Review by vyneer — CHANGES_REQUESTED** [review]
also the streamable clips dont get blurred for me, but im on mobile so maybe that has something to do with it

**Comment by vyneer** [comment]
i also noticed another clear mistake: i didnt win dgger of the year, please fix. thank u!

File: index.html (hunk 1 above)
Comment (ivxandx) [reply index.html:483]: ty for letting me know, emotes are replaced now, idk how to fix the streamable thing :( and it's 3am so I'll maybe just leave it like that (most of the clips aren't necessarily spoiled anyway)

also I have just reviewed dgger of the year and after throwing out all of the fraudulent ballots, it appears you have won with 100% of the vote

Comment (vyneer) [reply index.html:483]: yeah no worries, the streamable thing is a minor nitpick anyway

and pog, finally, after so many years FeelsStrongMan

**Review by vyneer — APPROVED** [review]

**Pull request merged**